include/log_handling.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * \brief Function Prototype Library to implement log handling
+ * \author jaspalsingh@cmail.carleton.ca
+ * \version 1.0
+ * \date 2019-12-05
+ */
+
+#ifndef LOG_HANDLING_H
+#define LOG_HANDLING_H
```

Comment:
Convention is for it to be:
__LOG_HANDLING_H__

Reply:
Updated in the latest fix